service/telemetry/config.go

```diff
@@ -150,7 +150,7 @@ func (sp *SpanProcessor) Unmarshal(conf *confmap.Conf) error {
 	}
 
 	if sp.Batch != nil {
-		if sp.Batch.Exporter.Console == nil {
+		if sp.Batch.Exporter.Console == nil && sp.Batch.Exporter.Otlp == nil {
```

Comment:
I think I commented on a different PR and cannot find it. But I will repeat here:
* This should be part of `Validate` and should be belong to the deepest object `sp.Batch.Exporter`.

Reply:
This is done now. I found another couple of validations (for the pull and period metric readers) which I moved into their own validate methods as well